recommend/runtimePolicy.go

```diff
@@ -113,14 +114,45 @@ func checkProcessFileData(sumResp []*opb.Response, distro string) *MatchSpec {
 	ms.Spec = pol.KubeArmorPolicySpec{
 		Action:   "Allow",
 		Message:  "serviceaccount access detected",
-		Tags:     []string{"KUBERNETES", "SERVICE ACCOUNT", "RUNTIME POLICY"},
+		Tags:     []string{"KUBERNETES", "SERVICE ACCOUNT", "RUNTIME POLICY", "MITRE_T1552.007_container_api", "MITRE"},
```

Comment:
This hardcoded policy rules in the code is not acceptable. Are we planning to handle it in different PR?

Reply:
@nyrahul we are having the runtime recommendation for serviceaccount as v1.
can we have something like the following
```
the user checkout the karmor summary and finds a list of destination files, and processes to secure. 
He uses karmor recommend --get-runtime "serviceaccount, .git, passwd" and in the backend, 
we split these keywords and check for them in logs using discovery-engine API and generate runtime policy
```
will this approach work?